common/templates/license.html

```diff
@@ -0,0 +1,55 @@
+<%page args="license, button=False, button_size='88x31'"/>
+## keep this synchronized with the contents of cms/templates/js/license-selector.underscore
+<%!
+from django.utils.translation import ugettext as _
+
+def parse_license(lic):
```

Comment:
@singingwolfboy and I discussed this. This is the only place that needs to parse the license file, so we agreed that it is okay in this instance to leave the function here.